internal/runners/branch/tree.go

```diff
@@ -0,0 +1,192 @@
+package branch
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/ActiveState/cli/pkg/platform/api/mono/mono_models"
+	"github.com/go-openapi/strfmt"
+)
+
+type branchNode struct {
+	Label string
+	ID    string
+}
+
+type tree map[branchNode]tree
+
+type BranchTree struct {
+	tree                  tree
+	branches              mono_models.Branches
+	rootBranches          mono_models.Branches
+	localBranch           string
+	branchFormatting      string
+	localBranchFormatting string
+}
+
+const (
+	edgeLink string = "│  "
+	edgeMid  string = "├─"
+	edgeEnd  string = "└─"
+)
+
+func NewBranchTree() *BranchTree {
+	return &BranchTree{tree: make(tree)}
+}
+
+func (bt *BranchTree) BuildFromBranches(branches mono_models.Branches) {
+	bt.rootBranches = getRootBranches(branches)
+	for _, branch := range bt.rootBranches {
+		bt.tree[branchNode{branch.Label, branch.BranchID.String()}] = buildBranchTree(branch, branches)
+	}
+}
+
+func buildBranchTree(currentBranch *mono_models.Branch, branches mono_models.Branches) tree {
+	t := getChildren(currentBranch, branches)
```

Comment:
As a rule of thumb I'd suggest naming your vars unless you're only using it within a ~2-3 line span.